static/app/components/profiling/flamegraph.tsx

```diff
@@ -299,6 +323,13 @@ function Flamegraph(props: FlamegraphProps): ReactElement {
             />
           </ProfileDragDropImport>
         }
+        frameStack={
+          <FrameStack
+            getFrameColor={getFrameColor}
+            formatDuration={flamegraph ? flamegraph.formatter : noopColor}
```

Comment:
why do you pass `noopColor` to `formatDuration`?

Reply:
should have been noopDuration. Renamed it